src/main/java/org/rocstreaming/roctoolkit/RocLogger.java

```diff
@@ -5,7 +5,7 @@
 
 public class RocLogger {
```

Comment:
Alternatively, we can configure jul to notify us when log level changes, and propagate this level to roc. In this case we don't need to set debug level unconditionally.

Reply:
fixed.

made setLevel private and RocLogger package-private.

not renamed to Logger because there will be conflicts with `java.util.logging.Logger`